### ISSUE TITLE
Crowdsec release 1.4.6 break running existing version 1.4.5

### ISSUE THREAD

**Issue by mathieuHa** [crowdsecurity/crowdsec#2053]
### What happened?

Crowdsec released 1.4.6 the 10th of february 2023.

My instance on 1.4.5 restarted the 11, it tried to fetch info from the HUB and crashed in loop because ressource does'nt exist.

```
time="11-02-2023 09:42:44" level=info msg="Machine 'crowdsec' successfully added to the local API"
time="11-02-2023 09:42:44" level=info msg="API credentials dumped to '/etc/crowdsec/local_api_credentials.yaml'"
Check if lapi needs to register an additional agent
time="11-02-2023 09:42:44" level=warning msg="Crowdsec is not the latest version. Current version is 'v1.4.5' and the latest stable version is 'v1.4.6'. Please update it!"
time="11-02-2023 09:42:44" level=warning msg="As a result, you will not be able to use parsers/scenarios/collections added to Crowdsec Hub after CrowdSec v1.4.6"
time="11-02-2023 09:42:44" level=fatal msg="Failed to get Hub index : failed to download index: bad http code 404 while requesting https://hub-cdn.crowdsec.net/v1.4.5/.index.json"
Regenerate local agent credentials
time="11-02-2023 09:42:49" level=info msg="Machine 'crowdsec' successfully added to the local API"
time="11-02-2023 09:42:49" level=info msg="API credentials dumped to '/etc/crowdsec/local_api_credentials.yaml'"
Check if lapi needs to register an additional agent
time="11-02-2023 09:42:49" level=warning msg="Crowdsec is not the latest version. Current version is 'v1.4.5' and the latest stable version is 'v1.4.6'. Please update it!"
time="11-02-2023 09:42:49" level=warning msg="As a result, you will not be able to use parsers/scenarios/collections added to Crowdsec Hub after CrowdSec v1.4.6"
time="11-02-2023 09:42:49" level=fatal msg="Failed to get Hub index : failed to download index: bad http code 404 while requesting https://hub-cdn.crowdsec.net/v1.4.5/.index.json"
```

In fact, the container is right, https://hub-cdn.crowdsec.net/v1.4.5/.index.json returns a 404.

Env: docker docker-compose
version: 1.4.5

### What did you expect to happen?

When Crowdsec release a new version, old one keep working

### How can we reproduce it (as minimally and precisely as possible)?

Run 1.4.5 ?

### Crowdsec version

<details>

```console
$ cscli version
1.4.5
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
$ uname -a
Linux ubuntu-2gb-hel1-1 5.15.0-60-generic

</details>


### Enabled collections and parsers

<details>

```console
$ cscli hub list -o raw
Container is restarting in loop, cant do that
```

</details>

### Acquisition config

<details>
```console
# On Linux:
$ cat /etc/crowdsec/acquis.yaml /etc/crowdsec/acquis.d/*


# On Windows:
C:\> Get-Content C:\ProgramData\CrowdSec\config\acquis.yaml
# paste output here
</details>


### Config show

<details>

```console
$ cscli config show
# paste output here
```

</details>

### Prometheus metrics

<details>

```console
$ cscli metrics
# paste output here
```

</details>

### Related custom configs versions (if applicable) : notification plugins, custom scenarios, parsers etc.

<details>

</details>

**Comment by Bromelioideae** [comment]
I am experiencing the same issue.

**Comment by blotus** [comment]
Hello,

Sorry about that.

The hub branch for 1.4.5 was not created automatically when we released 1.4.6.

I've created the branch, and the container should pick it up automatically next time it starts.

**Comment by mathieuHa** [comment]
Thank you @blotus
